packages/uniforms-material/src/RadioField.tsx

```diff
@@ -1,15 +1,15 @@
 import FormControlLabel from '@material-ui/core/FormControlLabel';
 import FormLabel from '@material-ui/core/FormLabel';
 import RadioMaterial, { RadioProps } from '@material-ui/core/Radio';
-import RadioGroup from '@material-ui/core/RadioGroup';
+import RadioGroup, { RadioGroupProps } from '@material-ui/core/RadioGroup';
 import React, { ReactNode } from 'react';
 import { FieldProps, connectField, filterDOMProps } from 'uniforms';
 
 import wrapField from './wrapField';
 
 export type RadioFieldProps = FieldProps<
   string,
-  RadioProps,
+  RadioProps & RadioGroupProps,
```

Comment:
As I can see in [`RadioGroup` API](https://material-ui.com/api/radio-group/), the `row` property is actually coming from the [`FormGroup`](https://material-ui.com/api/form-group/). I think it'd be better to add `row?: boolean` below rather than all `RadioGroupProps`, in order to limit incorrect props, e.g. `defaultValue` (they are not going to be used anyway as `...props` is being passed to `<Radio />`, not `<RadioGroup />`).

Reply:
out of curiosity why not support both `RadioProps` and `RadioGroupProps` interfaces as their own props like: 

```ts
export type RadioFieldProps = FieldProps<
  string,
  {
    allowedValues?: string[];
    checkboxes?: boolean;
    fullWidth?: boolean;
    helperText?: string;
    margin?: any;
    RadioProps: Pick<RadioProps, 'allowed' | 'props' | 'here'>; // HERE
    RadioGroupProps: Pick<RadioGroupProps, 'allowed' | 'props' | 'here'>; // HERE
    row?: boolean;
    transform?(value: string): string;
  }
>;
```

Then pass those on to the actual components. `material-ui` seems to follow a similar pattern (without the `Pick`) for `InputLabelProps` and `InputProps` on certain components.

Reply:
We can do that, but I don't think there's a need for that here - existing props cover all usable `RadioGroupProps` (with this PR merged). Also, we try to keep fields consistent across themes. If you think it'd make sense, do file your ideas in an issue to have this PR small and focused.